### PR TITLE
Add quick fix for S2688 ('use-isnan')

### LINKED ISSUE #3009: Add quick fix for S2688: "NaN" should not be used in comparisons
ESLint rule.

### PR DESCRIPTION
**Pull request by yassin-kammoun-sonarsource**
Fixes #3009

### CODE REVIEW

File: eslint-bridge/tests/rules/decorators/use-isnan-decorator.test.ts [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+    },
+  ],
+  invalid: [
+    {
+      code: `if (n == NaN) {}`,
+      errors: [
+        {
+          suggestions: [
+            {
+              desc: `Use "Number.isNaN()"`,
```
Comment (vilchik-elena) [inline eslint-bridge/tests/rules/decorators/use-isnan-decorator.test.ts:39]: I think `Number.isNaN` and `isNaN` is not the same function, so we can't use this message for such fix

File: eslint-bridge/src/rules/decorators/use-isnan-decorator.ts [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+        case '!=':
+        case '!==':
+          negate = true;
+          break;
+        case '==':
+        case '===':
+          negate = false;
+          break;
+      }
+      if (negate !== null) {
```
Comment (vilchik-elena) [inline eslint-bridge/src/rules/decorators/use-isnan-decorator.ts:45]: I don't get which test case covers this condition's else branch

**Review by vilchik-elena — CHANGES_REQUESTED** [review]

File: eslint-bridge/tests/rules/decorators/use-isnan-decorator.test.ts (hunk 1 above)
Comment (yassin-kammoun-sonarsource) [reply eslint-bridge/tests/rules/decorators/use-isnan-decorator.test.ts:39]: My bad, I thought they were the same. I will suggest both then.

File: eslint-bridge/src/rules/decorators/use-isnan-decorator.ts (hunk 2 above)
Comment (yassin-kammoun-sonarsource) [reply eslint-bridge/src/rules/decorators/use-isnan-decorator.ts:45]: There is none, but it's surprising to see 100% coverage.

**Review by vilchik-elena — APPROVED** [review]

**Pull request merged**